### PR TITLE
Feature/hive

### PR DESCRIPTION
**Pull request by Rizato**
Add a hive page to tell users how to connect.

### CODE REVIEW

File: _i18n/en/_docs/How-to-connect-to-HIVE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+For this next command you should run it in a separate tab, terminal, inside tmux., or any other means to leave this command running.
+
+```bash
+ssh -i /path/to/key -L 31337:hive.polyswarm.network:31337 <user>@gate.polyswarm.io
+```
+
+You will not see a prompt on that terminal, but don't worry, you have an open tunnel.
+
+With the open tunnel, connect to polyswarmd at `http://localhost:31337`
```
Comment (mrtizmoatwork) [inline _i18n/en/_docs/How-to-connect-to-HIVE.md]: I think a minor change would make this statement better. Something like:

> With the tunnel open, use your browser to connect to polyswarmd at ....

And just to make sure, is it "http" or "https"?

File: _i18n/en/_docs/How-to-connect-to-HIVE.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### Using polyswarmd
+
+Checkout the polyswarmd api on our docs [here](API-polyswarm)
```
Comment (mrtizmoatwork) [inline _i18n/en/_docs/How-to-connect-to-HIVE.md]: I'd revise this slightly to be

> Check out the [polyswarmd api docs](API-polyswarm).

File: _i18n/en/_docs/How-to-connect-to-HIVE.md (hunk 1 above)
Comment (Rizato) [reply _i18n/en/_docs/How-to-connect-to-HIVE.md]: We aren't connecting via the browser. They are connecting via a microengine, curl or whatever else. I will make that more clear.

Http is correct.

File: _i18n/en/_docs/How-to-connect-to-HIVE.md (hunk 2 above)
Comment (Rizato) [reply _i18n/en/_docs/How-to-connect-to-HIVE.md]: Will do.

Comment (myhndl) [reply _i18n/en/_docs/How-to-connect-to-HIVE.md]: this is addressed

File: _i18n/en/_docs/How-to-connect-to-HIVE.md (hunk 1 above)
Comment (myhndl) [reply _i18n/en/_docs/How-to-connect-to-HIVE.md]: nothing to do

**Pull request merged**